contracts/contracts/strategies/CurvePoolBooster.sol

```diff
@@ -0,0 +1,415 @@
+                totalRewardAmount: 0,
+                maxRewardPerVote: 0
+            }),
+            targetChainId,
+            additionalGasLimit,
+            votemarket
+        );
+
+        emit NumberOfPeriodsUpdated(extraNumberOfPeriods);
+    }
+
+    /// @notice Manage the reward per vote of the campaign
+    /// @dev This function should be called after the campaign is created
+    /// @param newMaxRewardPerVote New maximum reward per vote
+    /// @param bridgeFee Fee to pay for the bridge
+    /// @param additionalGasLimit Additional gas limit for the bridge
+    function manageRewardPerVote(
+        uint256 newMaxRewardPerVote,
+        uint256 bridgeFee,
+        uint256 additionalGasLimit
+    ) external nonReentrant onlyGovernorOrStrategist {
+        require(campaignId != 0, "Campaign not created");
+        require(newMaxRewardPerVote > 0, "Invalid reward per vote");
+
+        // Manage the campaign
+        ICampaignRemoteManager(campaignRemoteManager).manageCampaign{
+            value: bridgeFee
+        }(
+            ICampaignRemoteManager.CampaignManagementParams({
+                campaignId: campaignId,
+                rewardToken: rewardToken,
+                numberOfPeriods: 0,
+                totalRewardAmount: 0,
+                maxRewardPerVote: newMaxRewardPerVote
+            }),
+            targetChainId,
+            additionalGasLimit,
+            votemarket
+        );
+
+        emit RewardPerVoteUpdated(newMaxRewardPerVote);
+    }
+
+    /// @notice Close the campaign.
+    /// @dev This function only work on the L2 chain. Not on mainnet.
+    /// @dev The _campaignId parameter is not related to the campaignId of this contract, allowing greater flexibility.
+    /// @param _campaignId Id of the campaign to close
+    function closeCampaign(
+        uint256 _campaignId,
+        uint256 bridgeFee,
+        uint256 additionalGasLimit
+    ) external nonReentrant onlyGovernorOrStrategist {
+        ICampaignRemoteManager(campaignRemoteManager).closeCampaign{
+            value: bridgeFee
+        }(
+            ICampaignRemoteManager.CampaignClosingParams({
+                campaignId: campaignId
+            }),
+            targetChainId,
+            additionalGasLimit,
+            votemarket
+        );
+        emit CampaignClosed(_campaignId);
+    }
+
+    /// @notice calculate the fee amount and transfer it to the feeCollector
+    /// @return Balance after fee
+    function _handleFee() internal returns (uint256) {
+        // Cache current rewardToken balance
+        uint256 balance = IERC20(rewardToken).balanceOf(address(this));
+        require(balance > 0, "No reward to manage");
+
+        uint256 feeAmount = (balance * fee) / FEE_BASE;
+
+        // If there is a fee, transfer it to the feeCollector
+        if (feeAmount > 0) {
+            // Transfer the fee to the feeCollector
+            IERC20(rewardToken).transfer(feeCollector, feeAmount);
```

Comment:
Will we ever bribe with USDT? If so, we'll need to use `safeTransfer`.
If not, the Slither error can be fixed by adding this before the `transfer`
`// slither-disable-next-line unchecked-transfer unused-return`

Reply:
Fixed in this commit: https://github.com/OriginProtocol/origin-dollar/pull/2327/commits/6997f35b6796ad46f68c91e869179ef98724f058.